QUEST_LV_0100.tsv

```diff
@@ -877,8 +877,8 @@ QUEST_LV_0100_20150317_000872	If you don't mind, can you get Eyes of Madness fro
 QUEST_LV_0100_20150317_000873	I don't want those filthy things to be near me either.{nl}But, it's worse to face against those demons alone.
 QUEST_LV_0100_20150317_000874	Thanks.{nl}I will share them with the other brothers before the demons find out.
 QUEST_LV_0100_20150317_000875	You are indeed the revelator.{nl}Now, we can concentrate on protecting the altar.
-QUEST_LV_0100_20150317_000876	Don't worry, the barrier of the main entrance on the 1st floor can be offset by the power of the chapel. {nl}That's what the altars are for. {nl}
```

Comment:
Yep, I understand that part; It's just the way the term is phrase could throw some people off thinking altar and chapel are the same thing.